src/views/Create.vue

```diff
@@ -19,8 +19,8 @@ import SplitPane from '../components/PaneSplit.vue'
 import PaneRight from '../components/PaneRight.vue'
 import PaneLeft from '../components/PaneLeft.vue'
 import Footer from '../components/Footer.vue'
-import { onUnmounted } from 'vue'
-import { store } from '../store'
+import { defineAsyncComponent, onUnmounted } from 'vue'
```

Comment:
Why we import `defineAsyncComponent` ?

Reply:
I did not catch that, that is an unused import.